news/2025/2025-02-21-osumania-7k-world-cup-2025-finals-recap.md

```diff
@@ -0,0 +1,217 @@
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/-YSOZ_eWE4E" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Mappool recap
+
+<style>
+    .osu-md--news .audio-player {
+        max-width: inherit;
+    }
+</style>
+
+**[Download the pool here! (98 MB)](https://osu.ppy.sh/beatmaps/packs/P276)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2376804412&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+![Finals mappool table](/wiki/shared/news/2025-02-21-osumania-7k-world-cup-2025-finals-recap/mappool.jpg)
+
+Mappool recap
+
+---
+
+[![](/wiki/shared/news/2025-02-21-osumania-7k-world-cup-2025-finals-recap/NIGHTMARE_INVITATION_banner.jpg)](https://osu.ppy.sh/beatmapsets/2321254#mania/4973026)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/458/Songs/EBIMAYO%20-%20NIGHTMARE%20INVITATION.mp3">
+    </audio>
+</div>
+
+osu! the Original.
+
+[![](/wiki/shared/news/2025-02-21-osumania-7k-world-cup-2025-finals-recap/CuttingEdgeInsaneArtifact_banner.jpg)](https://osu.ppy.sh/beatmapsets/2321277#mania/4973089)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/220/Songs/Marmalade_butcher_-_CuttingEdgeInsaneArtifact.mp3">
+    </audio>
+</div>
+
+osu! the Original 2: Electric boogaloo.
+
+[![](/wiki/shared/news/2025-02-21-osumania-7k-world-cup-2025-finals-recap/Welkius_banner.jpg)](https://osu.ppy.sh/beatmapsets/2321296#mania/4973158)
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/183/Songs/Aoi_-_Welkius.mp3">
+    </audio>
+</div>
+
+osu! the Original 3: Origins.
+
+## Semifinals summary
+
+| Team A |  |  | Team B |
+| --: | :-: | :-: | :-- |
+| **Japan** ::{ flag=JP }:: | **7** | 0 | ::{ flag=PE }:: Peru |
+| **Philippines** ::{ flag=PH }:: | **7** | 0 | ::{ flag=ID }:: Indonesia |
+| **Philippines** ::{ flag=PH }:: | **7** | 6 | ::{ flag=JP }:: Japan |
+| **South Korea** ::{ flag=KR }:: | **7** | 2 | ::{ flag=CN }:: China |
+
+Following last week's trends, with only the best of the best teams remaining, these matches were not short of amazing either. Regardless of a match results, the level of play shown in last week's skirmishes was unmatched to any other week. The pool's (almost) final bump in difficulty didn't make things any easier for the players.
+
+With South Korea being the sole victor of the winners' bracket, they currently have the most advantage over their rivals. With the Philippines surviving losers' bracket after their win against Indonesia and their **very** narrow victory over Japan, they'll get to rematch China in Grand Finals. That's a match we'd suggest bearing witnessing to given the very heated rivalry between China and the Philippines!
```

Comment:
removed "very" because repetition

```suggestion
With South Korea being the sole victor of the winners' bracket, they currently have the most advantage over their rivals. With the Philippines surviving the losers' bracket after their win against Indonesia and their **very** narrow victory over Japan, they'll get to rematch China in Grand Finals. That's a match we'd suggest bearing witness to given the heated rivalry between China and the Philippines!
```